upgrade/rendering-changes/3-4-0.md

```diff
@@ -0,0 +1,37 @@
+---
+title: 3.4.0
+description: Handle the rendering changes in the 3.4.0 release of the Telerik UI for Blazor components.
+page_title: Rendering Changes in 3.4.0
+slug: changes-in-3-4-0
+position: 1
+---
+
+# Rendering Changes in 3.4.0
+
+## "k-state" CSS classes
+
+Previous to the 3.4.0 version we used `k-state-*` CSS classes to denote that the state of an HTML element happened. As part of the 3.4.0 release, we remove the `state` segment of that classes. 
+
+>caption A list of all changed "k-state" classes
+
+| Pre-3.4.0        | 3.4.0      |
+|------------------|------------|
+| k-state-focus    | k-focus    |
+| k-state-focused  | k-focused  |
+| k-state-hover    | k-hover    |
+| k-state-hovered  | k-hovered  |
+| k-state-selected | k-selected |
+| k-state-disabled | k-disabled |
+| k-state-invalid  | k-invalid  |
+| k-state-active   | k-active   |
+
+
+## Dialog
+
+* Added `k-dialog-wrapper` CSS class to the topmost HTML element.
```

Comment:
It will be good to mention which is the element (e.g. `div.k-dialog` if that's the one), so that a person can more easily see what new selectors will be possible to use.

Reply:
It is just a div.k-dialog-wrapper, no other classes, the topmost div

Reply:
```suggestion
* Added `k-dialog-wrapper` CSS class to the outermost HTML element, which holds the modal overlay and the dialog instance.
```